src/auth/auth.module.ts

```diff
@@ -18,7 +18,7 @@ import { JwtStrategy } from './auth.strategy';
     JwtModule.register({
       secret: 'dev',
       signOptions: {
-        expiresIn: '30s',
+        expiresIn: '60s',
```

Comment:
set expire as environment variable. 
make 24 hour to expire.

Reply:
done make it 24 hours as environment variable.